scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -1,6 +1,29 @@
-// Script assets have changed for v2.3.0 see
-// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
 
+function gauss(base, sd){
```

Comment:
please move into a separate file, this can be reused in various other places

Reply:
done